client/src/layouts/Footer.vue

```diff
@@ -2,26 +2,26 @@
   <div>
     <q-footer>
       <q-tabs>
-        <q-tab name="feed">
-          <router-link to="/Feed">
-            <q-icon name="videocam" size="1.5em"/>
-          </router-link>
-        </q-tab>
-        <q-tab name="add">
-          <router-link to="/AddPost">
+        <router-link to="/Feed">
```

Comment:
For second demo this is a good idea to display all lists via v-for (will be based on array of objects with next route and icon)
Moreover I believe that **q-tabs** requires **q-tab** to be a direct child component, but more curiously there is a **q-route-tab** component made specifically for this case

Reply:
Thanks, I am going to fix this!